docs/network_new/README.md

```diff
@@ -0,0 +1,84 @@
+These tasks are mostly setting up network resources for users, where a network resource is a subnet in the user's wireguard mesh.
+
+- **multi-nic setups**  
+When someone is a farmer, exploiting nodes somewhere in a datacentre, where the nodes have multiple NICs, it is advisable (though not necessary) to differentiate OOB traffic (like initial boot setup) from user traffic (as well the overlay network as the outgoing NAT for nodes for IPv4) to be on a different NIC. With these parameters, a user will have to make sure their switches are properly configured, more in docs later.
+
+- **registering and configurations**  
+Once a node has booted and properly initialized, registering and configuring the node to be able to accept workloads and their associated network configs, is a two-step process.  
+First, the node registers it's live network setup to the BCDB. That is : all NICs with their associated IP addresses and routes are registered so a farm admin can in a second phase configure eventual separate NICs to handle different kinds of workloads.
+In that secondary phase, a farm admin can then set-up the NICs and their associated IP's manually, so that workloads can start using them.
+- **farmer considerations**  
+
+
+## wireguard explanations
+
+- **wireguard as pointopoint links and what that means**  
+Wireguard is a special type of VPN, where every instance is as well server for multiple peers as client towards multiple peers. That way you can create fanning-out connections als receive connections from multiple peers, creating effectively a mesh of connections Like this :
+![like so](https://github.com/threefoldtech/zos/blob/master/specs/network/HIDDEN-PUBLIC.png)
+
+- **wireguard port management**  
+Every wireguard point (a network resource point) needs a destination/port combo when it's  publicly reachable. The destination is a public ip, but the port is the differentiator. So we need to make sure every nwetwork wireguard listening port is unique in the node wehere it runs, and can be reapplied in case of a node's reboot.
+ZOS registers the ports **already in use** to the BCDB, so a user can the pick a port that is not yet used.
+
+- **wireguard and hidden nodes**  
+Hidden nodes are nodes that are in essence hidden behind a firewall, and unreachable from the Internet to an internal net, be it as an IPv4 NATed host or an IPv6 host that is firewalled in any way, where it's impossible to have connection initiations form the Internet to the node.  
+As such, these nodes can only partake in a network as client-only towards publicly reachable peers, and can only initiate the connections themselves. (ref previous drawing).  
+To make sure connectivity stays up, the clients (all) have a keepalive towards all their peers so that communications towards network resources in hidden nodes can be established.
+
+## caveats
```

Comment:
This caveats paragraph is actually what the farmer needs to be aware of when configuring their farm network.
@andhartl can you check if this is enough for them or should we add more detail ?

We should also reference this paragraph from the index of the network documentation as `Farm network consideration`